register-plus-redux.php

```diff
@@ -299,7 +299,7 @@ class Register_Plus_Redux {
                             $additional_fields_exist = TRUE;
                             break;
                         }
-                        else if ( 'terms' === $meta_field['display'] ) { $this->rpr_set_terms_exist( TRUE ) }
+                        else { $this->rpr_set_terms_exist( TRUE ); }
```

Comment:
I think, actually, this entire line should be removed.  I'm not sure why we walk through the $redux_usermeta and consider this at all, considering further down, before the actual display of the terms we do the exact same thing.  Do you find anything wrong with the removal of line 302?

Reply:
You are right. Line removed.